### PR TITLE
[RuboCop] Update to 0.48.1

### PR DESCRIPTION
**Pull request by koic**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -321,6 +317,12 @@ Style/NumericPredicate:
     - 'lib/bundler/source/git/git_proxy.rb'
     - 'lib/bundler/source/path.rb'
 
+# Offense count: 192
+# Cop supports --auto-correct.
+# Configuration parameters: PreferredDelimiters.
+Style/PercentLiteralDelimiters:
```
Comment (segiddins) [inline .rubocop_todo.yml]: might want to enable this one and set them to what the project is currently using?

Comment (koic) [inline .rubocop_todo.yml]: Thanks for your review.

Sure. It seems good to enable Style/PercentLiteralDelimiters cop according to the default setting of RuboCop.
Https://github.com/bbatsov/rubocop/blob/v0.48.1/config/enabled.yml#L651-L654

I applied Style/PercentLiteralDelimiters cop to the project by running `rubocop -a --only Style/PercentLiteralDelimiters` and `rubocop --auto-gen-config`. That commit is 4f7ffab5fc42ad54b3e040aedaf8c933806104d7.

How about that?

**Comment by segiddins** [comment]
@indirect r?

**Comment by koic** [comment]
I resolved the merge conflict. I know that RuboCop 0.49.0 has been released, but this PR does not yet support it.

**Comment by segiddins** [comment]
@indirect r?

**Comment by indirect** [comment]
looking good 👍 @bundlerbot r+

**Comment by koic** [comment]
Thank you very much for the review.

**Comment by segiddins** [comment]
@bundlerbot retry

**Pull request merged**